zcash_client_sqlite/src/wallet/init.rs

```diff
@@ -383,18 +384,53 @@ mod tests {
                 LEFT OUTER JOIN sapling_tree_shards prev_shard
                     ON shard.shard_index = prev_shard.shard_index + 1
                 INNER JOIN scan_queue ON
+                    (scan_queue.block_range_start >= subtree_start_height AND shard.subtree_end_height IS NULL) OR
                     (scan_queue.block_range_start BETWEEN subtree_start_height AND shard.subtree_end_height) OR
                     ((scan_queue.block_range_end - 1) BETWEEN subtree_start_height AND shard.subtree_end_height) OR
                     (
                         scan_queue.block_range_start <= prev_shard.subtree_end_height
                         AND (scan_queue.block_range_end - 1) >= shard.subtree_end_height
-                    )
-                INNER JOIN wallet_birthday
-                WHERE scan_queue.priority > {}
-                AND scan_queue.block_range_end > wallet_birthday.height",
+                    )",
                 u32::from(st.network().activation_height(NetworkUpgrade::Sapling).unwrap()),
+            ),
+            // v_sapling_shard_unscanned_ranges
+            format!(
+                "CREATE VIEW v_sapling_shard_unscanned_ranges AS
```

Comment:
What happens to existing wallets that already contain this view (defined differently)? In general are we okay to arbitrarily change existing views in unreleased versions?

Reply:
Yes.